src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationTest.kt

```diff
@@ -1339,6 +1340,69 @@ class ApplicationTest : IntegrationTestBase() {
     }
   }
 
+  @Test
+  fun `Submit Temporary Accommodation application returns 200`() {
+    `Given a User`(
+      staffUserDetailsConfigBlock = {
+        withTeams(
+          listOf(
+            StaffUserTeamMembershipFactory().produce()
+          )
+        )
+      }
+    ) { submittingUser, jwt ->
+      `Given a User`(roles = listOf(UserRole.ASSESSOR), qualifications = listOf(UserQualification.PIPE, UserQualification.WOMENS)) { assessorUser, _ ->
```

Comment:
When we submit a TA application does it matter what the user role is? Are we tied into the CAS1 stuff for the moment?

Qualifications aren't applicable to us and won't be for the foreseeable. Not sure if we can remove this setup.